exchanges/exchanges.go

```diff
@@ -2153,23 +2153,11 @@ func (poloniex *PoloniexExchange) processTrade(tradeParams []interface{}) (*polo
 	if err != nil {
 		return nil, -1, fmt.Errorf("failed to convert poloniex orderbook update volume to float: %v", err)
 	}
-	binKey := int64(price * 1e8)
-	var existingOrder *poloniexOrder
-	var found bool
-	if direction == poloniexAskDirection {
-		existingOrder, found = poloniex.asks[binKey]
-	} else {
-		existingOrder, found = poloniex.buys[binKey]
-	}
-	if !found {
-		return nil, -1, fmt.Errorf("no order found to match reported poloniex trade")
-	}
-
-	order := &poloniexOrder{
+	trade := &poloniexOrder{
 		price:  price,
-		volume: existingOrder.volume - volume,
```

Comment:
This was wrong. I'm fairly certain that this signal is for trades that take place off of the order books.